Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionAPIRuntime.mm

```diff
@@ -1308,6 +1308,51 @@
     [manager run];
 }
 
+TEST(WKWebExtensionAPIRuntime, ConnectFromWebPageWithWrongIdentifier)
+{
+    auto *backgroundScript = Util::constructScript(@[
+        @"browser.runtime.onMessageExternal.addListener((message, sender, sendResponse) => {",
```

Comment:
is it intentional that we're not using optional chaining for the tests anymore?

Reply:
It depends. I think `browser.runtime?.onMessageExternal.addListener` would be best here.

Reply:
It is most important to do on results, when doing `test.assertEq(info?.foo, 42)`. If `info` is `null` that will fail otherwise.